allpairs/src/allpairs_master.c

```diff
@@ -64,6 +71,7 @@ static void show_help(const char *cmd)
 	fprintf(stdout, " %-30s Set the project name to <project>\n", "-N,--project-name=<project>");
 	fprintf(stdout, " %-30s Priority. Higher the value, higher the priority.\n", "-P,--priority=<integer>");
 	fprintf(stdout, " %-30s Enable debugging for this subsystem.  (Try -d all to start.)\n", "-d,--debug=<flag>");
+	fprintf(stdout, " %-30s Compute half of a symmetric matrix.\n", "   --symmetric");
```

Comment:
Is it a matrix? 
If not, I rather have --commutative, to mean that F(a,b) = F(b,a).